src/components/directorybrowser/directorybrowser.js

```diff
@@ -124,7 +124,7 @@ function getEditorHtml(options, systemInfo) {
         html += '<div class="results paperList" style="max-height: 200px; overflow-y: auto;"></div>';
     }
     if (options.enableNetworkSharePath) {
-        html += '<div class="inputContainer" style="margin-top:2em;">';
+        html += '<div id="networkPathContainer" class="inputContainer hide" style="margin-top:2em;">';
```

Comment:
is there a particular reason?
this can be achieved by passing `enableNetworkSharePath: false` on the options

Reply:
lol I didn't even notice that :man_facepalming: